config/edgeprobe_existing.conf

```diff
@@ -9,9 +9,6 @@
 
 // Override paths from params.conf, since these might point to paths on a
 // different system.
-global_ro_exp_dir = "/nfs/jsalt/share/exp/default"
-project_dir = ${JIANT_PROJECT_PREFIX}
-data_dir = ${JIANT_DATA_DIR}  // required - should point to data on NFS.
```

Comment:
How is this configured now? Directly with env vars?

Reply:
Oops—I missed that this wasn't inheriting from defaults.conf.